examples/deployments/uc_functions/run.py

```diff
@@ -0,0 +1,141 @@
+"""
+# Set environment variables
+export DATABRICKS_HOST="..."
+export DATABRICKS_TOKEN="..."
+export DATABRICKS_WAREHOUSE_ID="..."
```

Comment:
Can we add more explanation for each of these env vars: why are we setting it, what are some example values?